sdk/storage/azure-storage-queue/tests/_shared/testcase.py

```diff
@@ -310,13 +313,39 @@ def generate_sas_token(self):
     def generate_fake_token(self):
         return FakeTokenCredential()
 
+    def _get_service_version(self, **kwargs):
+        env_version = os.environ.get("AZURE_LIVE_TEST_SERVICE_VERSION", LATEST_SERVICE_VERSION)
+        return kwargs.pop("service_version", env_version)
```

Comment:
Is there a way to not copy code between storage modules? "storage test shared" module would be great.
We have one in .NET, we started one in Java while doing similar changes. Maybe we could use this opportunity to start one in python?

Reply:
We can probably put this in `devtools_testutils`, I'm working on an improvement PR for this that includes code we have shared in tables and storage [here](https://github.com/Azure/azure-sdk-for-python/pull/19040)

Reply:
That'd be great. not a blocker here, but we should do something about all these clones in long term.

Reply:
After merging this, I can make changes for that